app/models/order_process.rb

```diff
@@ -7,6 +7,8 @@ class OrderProcess < ApplicationRecord
   belongs_to :after_portfolio_item, :class_name => 'PortfolioItem'
   has_many :tag_links, :dependent => :destroy, :inverse_of => :order_process
 
+  validates :name, :uniqueness => {:scope => :tenant_id}
```

Comment:
can be `{:scope => :tenant}`
Do you want validate presence of name too?

Reply:
@bzwei Good question. I think we *can*, but I don't know if it's strictly necessary. I think I'll put in a separate PR for validating the presence of the name, since it makes sense, but don't think it should be tied to fixing this (small) bug.